Collision/src/main/java/dk/gruppeseks/bodtrd/collision/CollisionHandler.java

```diff
@@ -0,0 +1,331 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package dk.gruppeseks.bodtrd.collision;
+
+import dk.gruppeseks.bodtrd.common.data.Entity;
+import dk.gruppeseks.bodtrd.common.data.entityelements.Body;
+import dk.gruppeseks.bodtrd.common.data.entityelements.Body.Geometry;
+import static dk.gruppeseks.bodtrd.common.data.entityelements.Body.Geometry.CIRCLE;
+import static dk.gruppeseks.bodtrd.common.data.entityelements.Body.Geometry.RECTANGLE;
+import dk.gruppeseks.bodtrd.common.data.entityelements.Position;
+import dk.gruppeseks.bodtrd.common.data.util.Vector2;
+
+/**
+ *
+ * @author Chris & Morten
+ */
+public class CollisionHandler
+{
+
+    /**
+     * Check if 2 game objects are colliding.
+     *
+     * @param e1
+     * game object 1.
+     * @param e2
+     * game object 2.
+     * @return Returns true if the objects are colliding.
+     */
+    public static boolean isColliding(Position colPos, Body colBody, Entity e2)
```

Comment:
how about we pass in two Entities e1 and e2. create a new Object of type CollisionObject or whatever naming suits. which we pass all the necessary calculated variables. and pass references to this CollisionObject to methods internal to the CollisionHandler. it should improve readability and clarity and maybe even performance in edge cases